src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,55 @@
+.nav {
+  &__item {
+    display: inline-block;
+    margin-right: 20px;
+
+    &:last-child {
```

Comment:
I just realized; instead of using margin value twice, we can use it only once with;
```
&:not(:last-child)  { 
margin-right: 20px;
}
```
but it still works :)